<launch>
    <arg name="soma_config"/>
    <arg name="time_increment" default="1"/>
    <arg name="time_window" default="10"/>
    <arg name="periodic_cycle" default="10080"/>
    <arg name="trajectory_topic" default="/people_trajectory/trajectories/complete"/>
    
    <arg name="machine" default="localhost" />
    <arg name="user" default="" />

    <machine name="$(arg machine)" address="$(arg machine)" env-loader="$(optenv ROS_ENV_LOADER )" user="$(arg user)" default="true"/>

    <node pkg="people_temporal_patterns" type="counter_service.py" name="people_counter" respawn="false" output="screen">
        <param name="soma_config" value="$(arg soma_config)" type="string"/>
        <param name="time_increment" value="$(arg time_increment)" type="int"/>
        <param name="time_window" value="$(arg time_window)" type="int"/>
        <param name="periodic_cycle" value="$(arg periodic_cycle)" type="int"/>
        <param name="trajectory_topic" value="$(arg trajectory_topic)" type="string"/>
    </node>
</launch>
